app/js/settings.js

```diff
@@ -45,6 +45,7 @@ const defaultSettings = {
     "guitabbar": "always",
     "keeprecentlyclosed": true,
     "ignorecase": false,
+    "incsearch": false,
```

Comment:
Can we turn this on by default? What are your thoughts on this?

Reply:
I'm a big fan and apparently it's also enabled by default in neovim, so I've changed the default to `true`.